src/game/levels/__init__.py

```diff
@@ -12,12 +12,14 @@
 
 # Intialize Useful Constants
 PATH_TO_LEVELS: str = pathlib.Path(__file__).parent.absolute()
-PLATFORM_IMAGE_URL: str = Path(
+ASSET_PATH: str = Path(
     __file__
-).parent.parent.parent.absolute().__str__() + os.path.sep + "assets" + os.path.sep + "game" + os.path.sep + "GRASS_TILE.png"
-ENEMY_IMAGE_URL: str = Path(
-    __file__
-).parent.parent.parent.absolute().__str__() + os.path.sep + "assets" + os.path.sep + "game" + os.path.sep + "greek_soldier_walk1.png"
+).parent.parent.parent.absolute().__str__() + os.path.sep + "assets" + os.path.sep + "game" + os.path.sep
+PLATFORM_IMAGE_URL: str = ASSET_PATH + "wood_platform.png"
+HAY_IMAGE_URL: str = ASSET_PATH + "hay_bale.png"
+
+GROUND_IMAGE_URL: str = ASSET_PATH + "GRASS_TILE.png"
+ENEMY_IMAGE_URL: str = ASSET_PATH + "greek_soldier_walk1.png"
```

Comment:
I prefer string interpolation over string concat, but I doubt it makes any difference.